app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -8,6 +8,7 @@
     class="swagger-ui-wrap"
     data-url="<%= spec_url(@api_docs_service) %>"
     data-base-url="<%= base_url %>"
+    data-service-endpoint="<%= @api_docs_service.service&.proxy&.endpoint %>"
```

Comment:
The [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) says that one should not speak to strangers, and this is following a whole chain of strangers 😱

Reply:
You should at least delegate instead 😉

Reply:
@guicassolato suggested this approach
Any suggestion @Martouta ?

Reply:
Yeah. We'd better define a method in `ApiDocs::Service`.

Reply:
@damianpm yes, using delegate allowing nil. Perhaps also with the prefix. For example:
https://github.com/3scale/porta/blob/0a1016acf6876b578964251944d354e577617b9d/app/models/contract.rb#L50